docs/pages/eas-update/estimate-bandwidth.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Estimate bandwidth usage
+description: Learn how to estimate bandwidth usage with EAS Update.
+---
+
+# Understanding Update Bandwidth Usage
+
+EAS Update enables an app to update its own non-native pieces (such as JS, styling, and images) over-the-air. This guide explains how bandwidth is consumed and optimized.
+
+## Bandwidth Calculation Breakdown
+
+Each billing cycle includes a predefined bandwidth allocation, which translates to a specific number of updates your users can download. Here's how we estimate bandwidth usage:
+
+- **Update Size:** The key factor in bandwidth consumption is the size of each update. For example, if an update modifies only the JavaScript portion of your app and is **10 MB uncompressed**, we can further optimize this with compression.
+- **Compression Ratio:** Hermes bytecode, commonly used in React Native apps, achieves an estimated **2.6× compression ratio**. This means the actual downloaded update size is approximately:
+
+  ```text
+  10 MB / 2.6 ≈ 3.85 MB per update
+  ```
+
+- **Total Downloads:** Given a bandwidth allocation, we estimate how many users can download an update. For example, if your plan includes **1 TiB (1,024 GiB) of bandwidth per billing cycle**, the number of updates that can be downloaded is:
+
+  ```text
+  (1,024 GiB × 1,024 MiB) / 3.85 MiB ≈ 272,935 updates
+  ```
+
+## Measuring Your Actual Update Size
+
+To determine the actual compressed size of your Hermes bundle, run the following command:
+
+```sh
+gzip -9 -k bundle.hbc
+ls -lh bundle.hbc.gz
+```
+
+This will generate a **gzip-compressed** version of your Hermes bundle (`bundle.hbc.gz`) and display its size. You can use this to refine bandwidth calculations based on your app’s real-world update sizes.
```

Comment:
```suggestion
To determine the actual compressed size of your Hermes bundle, run the following commands:

```sh
brotli -5 -k bundle.hbc
gzip -9 -k bundle.hbc
ls -lh bundle.hbc.br bundle.hbc.gz
```

This will generate **Brotli-** and **Gzip-compressed** versions of your Hermes bundle (`bundle.hbc.br` and `bundle.hbc.gz`) and display their sizes. You can use this to refine bandwidth calculations based on your app’s real-world update sizes.
```